flask_appbuilder/templates/appbuilder/general/security/login_oauth.html

```diff
@@ -5,8 +5,9 @@
 
 <script type="text/javascript">
 
-var baseLoginUrl = {{url_for('AuthOAuthView.login')}};
-var baseRegisterUrl = {{url_for('AuthOAuthView.login')}};
+var baseLoginUrl = "{{url_for('AuthOAuthView.login')}}";
```

Comment:
This was interesting... we're missing the quotes, so this actually gets translated to:

```javascript
var baseLoginUrl = /login/;
```

Which works because it defines a regular expression that later gets cast to a string when it's concatenated to `currentSelection`!

Reply:
God...

Reply:
It's ok, I blame Javascript. :)

Reply:
Hello ,
I have following test  case scenario , is  it taken care in testing or not?

If some one logged in to FlaskAppBuilder using  twitter account and in between someone delete/disable his twitter account then  is  user still able to use FlaskAppBuilder app (if he did not logout manually)  or get Access Denied Error and  How  we are  validating user authenticity in running application.